tutorials/beginner/logs.md

```diff
@@ -6,11 +6,17 @@ layout: tutorial-beginner
 
 # Looking to the logs
 
-While using openHAB, we will need to look the the logs generated by the software.
+While using openHAB, we will need to look at the the logs generated by the software to see if there are any issues that need your attention.
 
-Two log files are going to interest us.
+Two log files are going to interest us. The two files are called openhab.log and events.log with the events.log file showing what is happening on the Openhab event bus like sensors changing states, and the openhab.log file which shows the output of all bindings and addons.
 
-To find them:
+To watch the logs in realtime with a openhabian based setup use the below linux command which can be done via SSH with a program called putty from a windows or mac machine. CTRL+C will close the stream. You can also use SAMBA/network shares to open or copy the file directly.
+
+tail -f /var/log/openhab2/openhab.log -f /var/log/openhab2/events.log
+
+Bindings can have the amount of information turned up or down on a per binding basis and this is done with the Karaf console. 
```

Comment:
I would suggest to use something like `deepth of information´ and `increased or decreased`